nuntium/templates/thread/to.html

```diff
@@ -4,9 +4,9 @@
 {% block content_inner %}
 
     <div class="person-profile">
-        <img class="person-profile__photo" src="http://placehold.it/128x128" width="128" height="128">
+        <img class="person-profile__photo" src="{% if person.image %}{{person.image}}{% else %}http://placehold.it/128x128{% endif %}" width="128" height="128">
```

Comment:
If the person *does* have an image, do we really want to distort it to 128x128, regardless of the original dimensions?

If they don't, we want a better placeholder than this. Even http://upload.wikimedia.org/wikipedia/en/b/b1/Portrait_placeholder.png would be better.